### ISSUE TITLE
Python3.4 and pandas 0.17, core dump on loading a csv

### ISSUE THREAD

**Issue by teto** [pydata/pandas#11549]
Hi,

I get a core dump while only loading a csv with d = pd.read_csv('roudrobin.csv')
Here is the csv:
https://transfer.sh/EYNLT/roudrobin.csv

 pd.__version__
Out[3]: '0.17.0'

```
gdb python3 core                                                                                     
GNU gdb (Ubuntu 7.10-1ubuntu2) 7.10
Copyright (C) 2015 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word"...

warning: bad breakpoint number at or near 'pretty-printers'
Source directories searched: /home/teto/iperf3/src:/home/teto/ns3off/src:/home/teto/iperf2/src:$:$cdir:$cwd
Reading symbols from python3...(no debugging symbols found)...done.

warning: core file may not match specified executable file.
[New LWP 24251]

warning: platform-specific solib_create_inferior_hook did not load initial shared libraries.
Reading symbols from /lib/x86_64-linux-gnu/libpthread.so.0...Reading symbols from /usr/lib/debug//lib/x86_64-linux-gnu/libpthread-2.21.so...done.
done.
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Reading symbols from /lib/x86_64-linux-gnu/libc.so.6...Reading symbols from /usr/lib/debug//lib/x86_64-linux-gnu/libc-2.21.so...done.
done.
Reading symbols from /lib/x86_64-linux-gnu/libdl.so.2...Reading symbols from /usr/lib/debug//lib/x86_64-linux-gnu/libdl-2.21.so...done.
done.
Reading symbols from /lib/x86_64-linux-gnu/libutil.so.1...Reading symbols from /usr/lib/debug//lib/x86_64-linux-gnu/libutil-2.21.so...done.
done.
Reading symbols from /lib/x86_64-linux-gnu/libexpat.so.1...(no debugging symbols found)...done.
Reading symbols from /lib/x86_64-linux-gnu/libz.so.1...(no debugging symbols found)...done.
Reading symbols from /lib/x86_64-linux-gnu/libm.so.6...Reading symbols from /usr/lib/debug//lib/x86_64-linux-gnu/libm-2.21.so...done.
done.
Reading symbols from /lib64/ld-linux-x86-64.so.2...Reading symbols from /usr/lib/debug//lib/x86_64-linux-gnu/ld-2.21.so...done.
done.
Reading symbols from /usr/lib/python3.4/lib-dynload/readline.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /lib/x86_64-linux-gnu/libreadline.so.6...(no debugging symbols found)...done.
Reading symbols from /lib/x86_64-linux-gnu/libtinfo.so.5...(no debugging symbols found)...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/hashtable.cpython-34m.so...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/numpy/core/multiarray.cpython-34m.so...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/numpy/core/umath.cpython-34m.so...done.
Reading symbols from /usr/lib/python3.4/lib-dynload/_ctypes.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /usr/lib/x86_64-linux-gnu/libffi.so.6...(no debugging symbols found)...done.
Reading symbols from /usr/lib/python3.4/lib-dynload/_bz2.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /lib/x86_64-linux-gnu/libbz2.so.1.0...(no debugging symbols found)...done.
Reading symbols from /usr/lib/python3.4/lib-dynload/_hashlib.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /lib/x86_64-linux-gnu/libcrypto.so.1.0.0...(no debugging symbols found)...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/numpy/linalg/lapack_lite.cpython-34m.so...(no debugging symbols found)...done.
Reading symbols from /usr/lib/liblapack.so.3...(no debugging symbols found)...done.
Reading symbols from /usr/lib/libblas.so.3...(no debugging symbols found)...done.
Reading symbols from /usr/lib/x86_64-linux-gnu/libgfortran.so.3...(no debugging symbols found)...done.
Reading symbols from /lib/x86_64-linux-gnu/libgcc_s.so.1...(no debugging symbols found)...done.
Reading symbols from /usr/lib/x86_64-linux-gnu/libquadmath.so.0...(no debugging symbols found)...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/numpy/linalg/_umath_linalg.cpython-34m.so...(no debugging symbols found)...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/numpy/fft/fftpack_lite.cpython-34m.so...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/numpy/random/mtrand.cpython-34m.so...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/tslib.cpython-34m.so...done.
Reading symbols from /usr/lib/python3.4/lib-dynload/_ssl.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /lib/x86_64-linux-gnu/libssl.so.1.0.0...(no debugging symbols found)...done.
Reading symbols from /usr/lib/python3.4/lib-dynload/_lzma.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /lib/x86_64-linux-gnu/liblzma.so.5...(no debugging symbols found)...done.
Reading symbols from /usr/lib/python3.4/lib-dynload/parser.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/lib.cpython-34m.so...done.
Reading symbols from /usr/lib/python3.4/lib-dynload/_csv.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/algos.cpython-34m.so...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/index.cpython-34m.so...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/_period.cpython-34m.so...done.
Reading symbols from /usr/lib/python3.4/lib-dynload/_decimal.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /usr/lib/x86_64-linux-gnu/libmpdec.so.2...(no debugging symbols found)...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/_sparse.cpython-34m.so...done.
Reading symbols from /usr/lib/python3.4/lib-dynload/_opcode.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/parser.cpython-34m.so...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/json.cpython-34m.so...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/msgpack/_packer.cpython-34m.so...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/msgpack/_unpacker.cpython-34m.so...done.
Reading symbols from /usr/lib/python3.4/lib-dynload/_json.cpython-34m-x86_64-linux-gnu.so...(no debugging symbols found)...done.
Reading symbols from /lib/x86_64-linux-gnu/libuuid.so.1...(no debugging symbols found)...done.
Reading symbols from /home/teto/.local/lib/python3.4/site-packages/pandas/_testing.cpython-34m.so...done.
Reading symbols from system-supplied DSO at 0x7ffec77b7000...(no debugging symbols found)...done.
Core was generated by `python3'.
Program terminated with signal SIGSEGV, Segmentation fault.
Reading in symbols for ../sysdeps/x86_64/strlen.S...done.
#0  strlen () at ../sysdeps/x86_64/strlen.S:106
106	../sysdeps/x86_64/strlen.S: Aucun fichier ou dossier de ce type.
(gdb) bt
Reading in symbols for pandas/parser.c...done.
#0  strlen () at ../sysdeps/x86_64/strlen.S:106
#1  0x00000000005a8f1c in PyBytes_FromString ()
#2  0x00007f58e4a7a50c in __pyx_f_6pandas_6parser__try_int64 (__pyx_v_parser=0x1979bf0, __pyx_v_col=__pyx_v_col@entry=23, 
    __pyx_v_line_start=__pyx_v_line_start@entry=1, __pyx_v_line_end=__pyx_v_line_end@entry=3522, 
    __pyx_v_na_filter=__pyx_v_na_filter@entry=1, __pyx_v_na_hashset=__pyx_v_na_hashset@entry=0x187a680) at pandas/parser.c:20849
#3  0x00007f58e4a87c47 in __pyx_f_6pandas_6parser_10TextReader__convert_with_dtype (__pyx_v_self=0x7f58e47a39e8, 
    __pyx_v_dtype=0x7f58e4cb5bc8, __pyx_v_i=23, __pyx_v_start=1, __pyx_v_end=3522, __pyx_v_na_filter=1, __pyx_v_user_dtype=0, 
    __pyx_v_na_hashset=0x187a680, __pyx_v_na_flist=0x7f58ece2d9e8) at pandas/parser.c:12372
#4  0x00007f58e4a8beaf in __pyx_f_6pandas_6parser_10TextReader__convert_tokens (__pyx_v_self=__pyx_v_self@entry=0x7f58e47a39e8, 
    __pyx_v_i=__pyx_v_i@entry=23, __pyx_v_start=__pyx_v_start@entry=1, __pyx_v_end=__pyx_v_end@entry=3522, 
    __pyx_v_name=__pyx_v_name@entry=0x7f58e3ee93f0, __pyx_v_na_filter=__pyx_v_na_filter@entry=1, __pyx_v_na_hashset=0x187a680, 
    __pyx_v_na_flist=0x7f58ece2d9e8) at pandas/parser.c:11744
#5  0x00007f58e4a8ee50 in __pyx_pf_6pandas_6parser_10TextReader_16_convert_column_data (__pyx_v_footer=<optimized out>, 
    __pyx_v_upcast_na=<optimized out>, __pyx_v_rows=<optimized out>, __pyx_v_self=0x7f58e47a39e8) at pandas/parser.c:10858
#6  __pyx_pw_6pandas_6parser_10TextReader_17_convert_column_data (__pyx_v_self=0x7f58e47a39e8, __pyx_args=<optimized out>, 
    __pyx_kwds=<optimized out>) at pandas/parser.c:10148
#7  0x00007f58e4a78b80 in __Pyx_PyObject_Call (kw=0x7f58eb91b9c8, arg=<optimized out>, func=0x7f58e4760548) at pandas/parser.c:32928
#8  __pyx_f_6pandas_6parser_10TextReader__read_rows (__pyx_v_self=0x7f58e47a39e8, __pyx_v_rows=<optimized out>, __pyx_v_trim=0)
    at pandas/parser.c:9465
#9  0x00007f58e4a67910 in __pyx_f_6pandas_6parser_10TextReader__read_low_memory (__pyx_v_self=0x7f58e47a39e8, 
    __pyx_v_rows=0x9ef5a0 <_Py_NoneStruct>) at pandas/parser.c:8338
#10 0x00007f58e4a6e3b4 in __pyx_pf_6pandas_6parser_10TextReader_8read (__pyx_v_rows=<optimized out>, __pyx_v_self=0x7f58e47a39e8)
    at pandas/parser.c:8082
#11 __pyx_pw_6pandas_6parser_10TextReader_9read (__pyx_v_self=0x7f58e47a39e8, __pyx_args=<optimized out>, __pyx_kwds=<optimized out>)
    at pandas/parser.c:8042
#12 0x0000000000511a1a in PyEval_EvalFrameEx ()
#13 0x0000000000519757 in PyEval_EvalCodeEx ()
#14 0x0000000000516213 in PyEval_EvalFrameEx ()
#15 0x0000000000519757 in PyEval_EvalCodeEx ()
#16 0x0000000000516213 in PyEval_EvalFrameEx ()
#17 0x0000000000515c2f in PyEval_EvalFrameEx ()
```

**Comment by jreback** [comment]
this is the same as #11374 and fixed for 0.17.1

**Comment by teto** [comment]
I upgraded pandas to HEAD and that indeed fixed it thanks !

2015-11-08 17:25 GMT+01:00 Jeff Reback <notifications@github.com>:
